extensions/amp-ad/0.1/amp-ad-xorigin-iframe-handler.js

```diff
@@ -523,6 +529,16 @@ export class AmpAdXOriginIframeHandler {
       this.intersectionObserver_.fire();
     }
   }
+
+  /**
+   * @param {!Error} error
+   * @param {string} message
+   */
+  userErrorForAnalytics(error, message) {
+    if (isUserErrorMessage(message)) {
```

Comment:
I think they should all be user error. Also your `isUserErrorMessage()` won't work here, since you've never added `USER_ERROR_EMBED_SENTINEL` to the error message.